lib/Core/SQLiteBuildDB.cpp

```diff
@@ -77,7 +78,7 @@ class SQLiteBuildDB : public BuildDB {
   /// * 6: Added `ordinal` field for dependencies.
   /// * 5: Switched to using `WITHOUT ROWID` for dependencies.
   /// * 4: Pre-history
-  static const int currentSchemaVersion = 14;
+  static const int currentSchemaVersion = 15;
```

Comment:
We need to coordinate merging this PR with https://github.com/apple/swift-llbuild/pull/832

Reply:
Owen mentioned some support tools might need to be updated to accommodate database schema changes.

Reply:
Changed this to 16 and this is resolved now.